commands/build.go

```diff
@@ -174,9 +174,8 @@ func runBuild(cmd *cobra.Command, args []string) error {
 			return fmt.Errorf("could not pull templates from function yaml file: %s", err.Error())
 		}
 	} else {
-		templateAddress := getTemplateURL("", os.Getenv(templateURLEnvironment), DefaultTemplateRepository)
-		if pullErr := pullTemplates(templateAddress); pullErr != nil {
-			return fmt.Errorf("could not pull templates for OpenFaaS: %v", pullErr)
+		for _, function := range services.Functions {
+			runTemplateStorePull(cmd, []string{function.Language})
```

Comment:
@alexellis Just informing you that I'm a little busy with my exams this week, so I'll be able to test the scenarios after this week.